backend/src/main/resources/application-dev.yml

```
@@ -20,4 +20,11 @@ cookie:
   sameSite: None
   access-token-path: /
-  refresh-token-path: /
\ No newline at end of file
+  refresh-token-path: /
+
+oci:
+  profile: INSTANCE_PRINCIPAL
+  region: ${REGION}
+  namespace: ${NAMESPACE}
+  bucketName: ${BUCKETNAME}
+  compartmentId: ${COMPARTMENTID}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)